### PR TITLE
Refactor the doc structure 

### PR DESCRIPTION
**Pull request by kayYOLO**

### CODE REVIEW

File: doc/references/python/python.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Overview  
 
-[Clicknium python package](https://pypi.org/project/clicknium/) supports to automate various types of applications, such as **Web** browser, **Windows Desktop** application, **Java** application and **SAP** windows Gui app, etc.
+[Clicknium Python package](https://pypi.org/project/clicknium/) supports to automate various types of applications, such as **Web** browser, **Windows** application, **Java** application and **SAP** Windows GUI App, etc.
```
Comment (Luluyu2022) [inline doc/references/python/python.md:3]: supports  automating...

**Review by Luluyu2022 — COMMENTED** [review]
updated till 72: doc/references/python/uielement/mouse_down.md

doc/concepts/recorder/[capture_similar_elements.md](http://capture_similar_elements.md/)
this .md has updated with xujinhua last week.

File: doc/references/python/exceptions/unreachablebrowserextensionerror.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-A string representing all the calls prior to the function that raised the exception.
-
-### browser_type
-- type: str
-
+# UnreachableBrowserExtensionError
+
+- extends: [WebError](./weberror.md)
+
+**UnreachableBrowserExtensionError is raised when the specified browser's extension is not in ready state. Please check if the extension is installed and enabled.**
```
[the remark this answers is not in the document]
Comment (kayYOLO) [reply doc/references/python/exceptions/unreachablebrowserextensionerror.md:5]: change to "Please make sure that the extension is installed and enabled"

File: doc/tutorial/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- [Clicknium VS Code Extension](./../quickstart.md#setup-clicknium-visual-studio-code-extension): Set up the development tool for coding automation script.
+- [Clicknium Automation Extension](./../concepts/extensions/extensions.md): To get better performance, Clicknium provides some extension targeted to different environment and senario. Developers can install extension accroding to their need.  
+Please follow [Quick Start](./../quickstart.md) to finish the installation.
+
+
+
+# Clicknkum Architecture Overview 
+Clicknium is a platform that lets you build your automation code. [Clicknium Python Module](./../quickstart.md#install-clicknium-python-package) will help to init ***Local Locator Store*** and ***Virtual Python Environment***. ***The VS Code Extension*** provides the Recorder to capture locaters and store it into Cloud/Local locater store. It also supports ***Code InteliSense*** to give a seamlessly coding experience. ***The Clicknium Cloud*** gives a chance to save and manage locators in the cloud and share locaters cross projects/machines.
```
[the remark this answers is not in the document]
Comment (Luluyu2022) [reply doc/tutorial/tutorial.md:18]: init---initialize
locater---locator
store them into the Cloud/Local locator store.

**Review by Luluyu2022 — COMMENTED** [review]
grammers and typo updates

File: doc/tutorial/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- [Clicknium VS Code Extension](./../quickstart.md#setup-clicknium-visual-studio-code-extension): Set up the development tool for coding automation script.
+- [Clicknium Automation Extension](./../concepts/extensions/extensions.md): To get better performance, Clicknium provides some extension targeted to different environments and senarios. Developers can select the needed extension to install.  
+Please follow [Quick Start](./../quickstart.md) to finish the installation.
+
+# Clicknium Architecture Overview 
+Clicknium is a platform where you can build your automation code. [Clicknium Python Module](./../quickstart.md#install-clicknium-python-package) will help to initialize ***Local Locator Store*** and ***Virtual Python Environment***. ***The VS Code Extension*** provides the Recorder to capture locators and store them into Cloud/Local Locator Store. It also supports ***Code InteliSense*** to give a seamlessly coding experience. ***The Clicknium Cloud*** gives a chance to save and manage locators in the cloud and share locaters cross projects/machines.  
+![Clicknium Arc](./../img/Clicknium_arc.png)
```
Comment (robinbob) [inline doc/tutorial/tutorial.md:17]: misspelling: locator

File: doc/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+## Automation Sample
+
+Get a sample script showing automation of web page and notepad.
+
+### Install and enable browser extension
+- Click `CLICKNIUM EXPLORER` in Visual Studio Code Activity Bar.  
+- Show `AUTOMATION EXTENSIONS` view in Visual Studio Code Side Bar.  
+- Choose the Edge browser and click the `Install` button.  
+- Open Edge browser, type `edge://extensions/` into address bar and enable `Clicknium Recorder` extension as below.
```
[the remark this answers is not in the document]
Comment (kayYOLO) [reply doc/quickstart.md:46]: it's two different step.

File: doc/tutorial/firstautomation.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+if __name__ == "__main__":
+    main()
+```
+
+## Capture first UI locator
+In the next step, we need to use [Locators](./../concepts/locator.md) to identity the UI element that we want to operate. [Clicknium Recorder](./../concepts/recorder/recorder.md) will help to catpure locators. 
+
+- Click `Capture` button in `LOCATORS` tab of `Explorer` side bar or pressing `Ctrl+F10 for shortcut in VS Code.    
+- [Clicknium Recorder](./../concepts/recorder/recorder.md) will pop up. Choose Chrome for capture technology.
```
Comment (robinbob) [inline doc/tutorial/firstautomation.md:27]: "Choose Chrome for capture technology" is not necessary, leave "auto detect" as is.

File: doc/concepts/locatorstore.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+## Local Locator Store
+Local locator Store will auto generate by default after the recorder captured first locator. Local locator store is a file in the project root path that store the locators. According to the implementation, the store will go with the Python project and it cann't be share to another project. On the other hand, it also means that the code can run offline. 
+
+![local locator store](./../img/locallocaterstoreinvs.png)
+
+## Cloud Locator Store 
+Cloud Locator Store is powered by Clicknium. After a successful log-in, cloud locator store will be connected. It can be referenced into the current project. The frequently-used locators are approperiated to store in cloud locator store so that can be shared in different projects, event different person. The Python script can run anywhere with Internet access. 
+
+### How to choose
```
Comment (robinbob) [inline doc/concepts/locatorstore.md]: Better change to a table. The format is hard to read. "Pros" is at same level of "How to choose"

File: doc/tutorial/locatorstore.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 2
+sidebar_label: Locator Store
+---
+# Locator Store
```
Comment (robinbob) [inline doc/tutorial/locatorstore.md:5]: the right side navigation panel is not right

File: doc/concepts/locatorstore.md (hunk 7 above)
Comment (kayYOLO) [reply doc/concepts/locatorstore.md]: changed

**Review by robinbob — APPROVED** [review]

**Pull request merged**